auth/src/test/java/com/redhat/lightblue/rest/auth/ldap/LdapRoleProviderIntegrationTest.java

```diff
@@ -102,4 +103,11 @@ public void testUserWhoDoesNotExist() throws Exception {
         assertTrue(provider.getUserRoles("idontexist").isEmpty());
     }
 
+    @Test
+    public void testHealthCheck() throws Exception {
+        
+        RolesProviderHealth health = provider.checkHealth();
+        System.out.println(health.details());
+        assertTrue(health.isHealthy());
+    }
```

Comment:
Is it possible to take the in memory ldap server down and do the health check to see it's unhealthy? That'd be pretty rad :-).